### PR TITLE
Add reverse triggers for docker jobs

### PR DESCRIPTION
**Pull request by JPWKU**
This will serialize the flow artifacts
from java merge jobs to docker jobs.

JIRA: RELENG-468
Signed-off-by: Jeremy Phelps <jphelps@linuxfoundation.org>

### CODE REVIEW

**Comment by trcox** [comment]
@JPWKU Just so that I make sure that I am understanding this, the intent here is the following flow?

PR commits -> triggers verify-java and verify-docker checks, both of which must pass

PR accepted/merged -> triggers merge-java -> (upon completion) triggers verify-docker with new artifacts -> (upon completion) triggers merge-docker to update the snapshot Nexus3 Docker images?

**Comment by JPWKU** [comment]
@trcox Basically yes...so we don't run into the situation where a java merge and a docker merge happen simultaneously.  What could happen there is the docker merge uses an older artifact than the java merge is building.

**Comment by JPWKU** [comment]
I _think_ this takes care of that but I'm double checking nuances.

File: jjb/edgex-templates-docker.yaml [hunk 1]
```diff
@@ -215,3 +217,5 @@
 
     triggers:
       - timed: 'H 11 * * *'
+      - reverse:
+          jobs: '{project-name}-{stream}-release-version-java-daily-no-sonar'
```
Comment (zxiiro) [inline jjb/edgex-templates-docker.yaml]: Should remove the timed here since java-daily is already timed.

Comment (JPWKU) [reply jjb/edgex-templates-docker.yaml]: Fixed.

**Review by zxiiro — APPROVED** [review]

**Pull request merged**